opm/input/eclipse/share/keywords/900_OPM/C/CSTRESS

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "CSTRESS",
+  "sections": [
+    "GRID"
+  ],
+  "description": "The CSTRESS item is used to set the critcal stress  in mechanics models.",
+  "data": {
+    "value_type": "DOUBLE",
+    "dimension": "1"
+  }
+}
```

Comment:
eol at eof

Reply:
> eol at eof

Good catch.  I've pushed an update to address this.